appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -19,14 +21,16 @@ export class CaseDetailsPage extends Page {
 		readyToStart: 'ready-to-start',
 		issueDetermination: 'issue-determination',
 		addLinkedAppeal: 'add-linked-appeal',
+		addRelatedAppeals: 'add-related-appeals',
 		uploadFile: '#upload-file-1',
 		changeAppealType: 'change-appeal-type',
 		addAgreementToChangeDescriptionEvidence: 'add-agreement-to-change-description-evidence',
 		manageAgreementToChangeDescriptionEvidence: 'manage-agreement-to-change-description-evidence',
 		addCostsDecision: 'add-costs-decision',
 		changeSiteOwnership: 'change-site-ownership',
 		changeLpaqDueDate: 'change-lpa-questionnaire-due-date',
-		changeStartDate: 'change-start-case-date'
+		changeStartDate: 'change-start-case-date',
+		getAppealStartDate: '.appeal-start-date > .govuk-summary-list__value'
```

Comment:
This isn't quite the right place to add this selector as the collection is `_cyDataSelectors` so it should only have `data-cy` selectors in it

In this instance we can either
1) Have the selector inline on the actual `elements.getAppealStartDate` (which appears to already have been done)
2) Create a new collection for non-data-cy selectors

I'd suggest we just go with 1 for now to keep things simplier, which would just mean deleting this line

Reply:
Ah yes, ok have removed from the data cy selectors section